src/app/app-common/PageSource/include/OpenKneeboard/PlainTextPageSource.h

```diff
@@ -66,18 +69,21 @@ class PlainTextPageSource final : public IPageSource {
   std::vector<std::vector<winrt::hstring>> mCompletePages;
   std::vector<winrt::hstring> mCurrentPageLines;
```

Comment:
Given my notes about `std::string_view` to avoid copying, using `winrt::hstring` here because:
- there's no duplication between messages/current page lines/complete pages
- it's UCS-2, whereas the std::string and std::string_view are UTF-8